### PR TITLE
Editing in VSTS

### PR DESCRIPTION
**Pull request by Blackbaud-TommyVernieri**
Tutorial describing how to make edits in a web browser using VSTS.

Added info on Getting Started step 5.

### CODE REVIEW

**Comment by Blackbaud-TommyVernieri** [comment]
Here are a few note after some user testing.

* The way "path to page" is expressed is a little confusing. maybe split the "src/app" and path to page parts out so they're not styled the same way.
* It'd help to move the repository image up into the bulleted list where it's referenced.
* The "branch dropdown" button should be described using the word "master" since that's what will probably be displayed there.
* Clarify that when adding a new branch, you need to first click the New branch button, it's easy to think that you're supposed to be typing a new branch name into the branch filter.
* Provide a bit more guidance on branch name style (lower case, words separated by hyphens, short, preceded by "ft-" or "feature-").
* "browswer" typo.
* Establish some context for what it means to be "in" the new branch. I.e. a the new branch is complete copy of the master branch.
* Split apart the create PR step and the complete merge step.
* Describe how to get back to the files to make more changes after creating the PR.
* Describe how to check on the progress of the build while waiting for the comment to be added to the PR.
* Discuss pros and cons to making a bunch of changes then creating a PR vs creating a PR after the first change and iterating.
* Maybe add a brief description of how to add markdown to a page (or a basic template for new pages).
* Describe how to get out of local dev mode.
* Note to leave delete branch checked.
* Describe the difference between Complete and Set auto-complete.

**Comment by Blackbaud-LindseyRix** [comment]
Can you covert the `` to `<stache-image>`.  If you'd rather me make the change, just let me know.

**Comment by Blackbaud-TommyVernieri** [comment]
@Blackbaud-LindseyRix I'm going to shift to some other work for a bit. I've added you, Brandon, and Brandon as collaborators in that branch. I'd love for you to make the `` change or any other improvements to better align the page with Stache style or to improve the readability/usefulness of the content.

**Comment by Blackbaud-LindseyRix** [comment]
Cool, will do. Just didn't want to step on your toes.

File: src/app/learn/basics/vsts-editor/index.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
+<stache
+  pageTitle="Editing in VSTS"
```
Comment (Blackbaud-BrandonJones) [inline src/app/learn/basics/vsts-editor/index.html]: I would remove the `pageTitle` attribute here becuase the Header in the tutorial should take care of giving the page a title.

Should we consider making this `layout="container"` to match it's parent's layout? `basics`.

**Comment by Blackbaud-BrandonJones** [comment]
This is brilliant! Thank you so much for contributing this, it will go a LONG way in helping us with a requested feature :) `edit in ____` button.

**Pull request merged**